Cargo.lock

```diff
@@ -2137,18 +2137,6 @@ dependencies = [
  "untrusted 0.6.2 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
-[[package]]
```

Comment:
Was this on purpose?

Reply:
nope, don't know how that slipped in there :P